components/Dialogs/src/MDCDialogTransitionController.m

```diff
@@ -18,13 +18,24 @@
 
 @implementation MDCDialogTransitionController
 
-static const NSTimeInterval MDCDialogTransitionDuration = 0.27;
+static const NSTimeInterval MDCDialogDefaultTransitionDuration = 0.27;
```

Comment:
Nit: [Static constants should start with `k` and should not include project-specific prefixes](http://google.github.io/styleguide/objcguide.html#constants).  Suggestion: `kDefaultTransitionDuration`.